### DISCUSSION TITLE
A warning that occurs when a mechanical inhibition is set

### DISCUSSION THREAD

**Discussion by ytg922** [Open-Systems-Pharmacology/Forum#1358] [Questions & Problems] [answered]
Dear OSP community，I recently started running simulations after defining the mechanistic inhibitory processes of the compounds. However, in the processes interface, this warning always appears：
  
![image](https://user-images.githubusercontent.com/103750318/197436691-b998f909-599a-4f55-ab6e-e0b4ba22ebc2.png)
“Please note that for the mechanism-based inactivator no clearance process is defined via the inactivation process by default. In theory, for every inactivated enzyme molecule, also one inactivator molecule is cleared; this must be separately defined by the user in form of additional metabolization/excretion processes for the inhibitor.”
I've tried many things, but I don't know what this warning means? If there is a master know, please reply.
                                                                                                                                                     Tony

**Comment by AndreDlm** [comment, marked as answer]
Hi Tony, 
what this message means it that when your drug binds as inhibitor (`Ligand A` in the figure below) to the active site of the enzyme, the latter is inactivated. Note however, that the drug itself remains bound to the inactivated enzyme and does not dissociate. Hence, the inactivation of the enzyme also removes the drug molecule from the system; it is thus cleared.
However, the implementation of mechanism-based inhibition in PK-Sim does not automatically account for the removal of the inhibitor from the system. This may need to be added explicitly as separate clearance process.
![image](https://user-images.githubusercontent.com/26541603/197572138-3252a2e9-69c6-45cc-8a23-1b8e9a664ba9.png)
Figure from: https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7557591/

I hope this makes it clear,
André

**Reply by ytg922** [reply]
Thanks André, your answer was great！

**Comment by tobiasK2001** [comment]
Dear Tony,

there might be also cases were you can ignore this warning. Please check out this discussion:
https://github.com/Open-Systems-Pharmacology/Forum/issues/226

Best, Tobias

**Reply by ytg922** [reply]
Copy that, Mr. Tobias
